src/deprecation.js

```diff
@@ -5,6 +5,26 @@ var core = require('./core'),
     extras = require('./extras'),
     filters = require('./filters');
 
+// provide method to give a stack track for warnings
+// useful for tracking-down where deprecated methods/properties/classes
+// are being used within the code
+function warn(msg) {
+    var stack = new Error().stack; 
+
+    // chop off the stack trace which includes pixi.js internal calls
+    stack = stack.split('\n').splice(3).join('\n');
```

Comment:
It's at least 3 calls always, the `console.warn`, the inline `warn` function and the deprecation call below where `warn()` is invoked. Its removes those 3 calls.